internal/dcache/gc/gc.go

```diff
@@ -76,8 +95,34 @@ func Start() {
 		numGcWorkers: 100,
 		// Allow more requests to be queued than workers.
 		deletedFileQueue: make(chan *gcFile, 1000),
+		done:             make(chan struct{}),
+		//
+		// The deleteTimeOut is kept to 5 minutes, The time taken to delete a file that was scheduled in worst case is:
+		// (((stripeSize / chunkSize) * numReplicas) RPC calls + timeTaken to delete Metadata file + clustermap Update) * (Queue Size / numWorkers) for GC
+		// (((16M/4M) * 3) 100ms for each RPC call + 300ms + 300ms) * (1000 / 100) = 18000ms in worst case maybe.
+		// The time taken to delete the last file that was scheduled into the GC queue in worst case would be ~18s
```

Comment:
this is not accurate since the time taken by the RemoveChunk RPC will depend on the number of chunks, which can be large for large files.